digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ViiteTierekisteriClient.scala

```diff
@@ -0,0 +1,159 @@
+package fi.liikennevirasto.viite
+import java.util.Properties
+
+import fi.liikennevirasto.digiroad2.util.TierekisteriAuthPropertyReader
+import fi.liikennevirasto.viite.dao.ProjectRoadAddressChange
+import org.apache.http.client.methods.{HttpGet, HttpPost}
+import org.apache.http.entity.{ContentType, StringEntity}
+import org.apache.http.impl.client.HttpClientBuilder
+import org.json4s.jackson.Serialization
+import org.json4s.{DefaultFormats, StreamInput}
+import org.joda.time.format.DateTimeFormat
+import org.json4s.jackson.JsonMethods.parse
+
+import scala.util.control.NonFatal
+case class ChangeProject(id:Long, name:String, user:String, ely:Long, change_date:String, change_info:Seq[ChangeInfoItem])
+case class TRProjectStatus(id:Option[Long], id_tr_projekti:Option[Long], projekti:Option[Long], tunnus:Option[Long],
```

Comment:
Get rid of Finnish field names, use 
case class TRProjectStatus(id: Option[Long], ..., @JsonProperty("projekti") projectId: Option[Long], ...